app/presenters/curation_concerns/generic_work_show_presenter.rb

```diff
@@ -254,6 +254,15 @@ def field_value field, options = {}
       field_values(field_config, options)
     end
 
+    # The string NOTES, in all caps, followed by an optional colon,
+    # needs to be on its own line in the Provenance field.
+    # The regular expression also captures extra blank space
+    # before and after the match, as well as before and after the word "NOTES".
+    def split_provenance
+      return [nil, nil] if provenance.nil? || provenance == []
+      return ["",  nil] if provenance == ""
+      provenance.first.split(/\s*\n\n\s*NOTES:?\s*\n\n\s*/m)
```

Comment:
nice-looking regexp. Did you find "/m" was necessary?  I'm not sure it is?

Also not sure if we need `\r?\n` for every `\n` to get any potential Windows-style newlines that are `\r\n`.

Reply:
/m is multiline.
Good call re: windows newlines.
I'm leaning towards `/\s*\n\s*NOTES:?\s*\n\s*/m` since `\s` captures `\r`; let me see how that pans out.

Reply:
I think _maybe_ the only effect of the multiline flag is to have `.` match newlines. This regex doesn't have any `.` in it, let alone ones that are supposed to match newlines. I think maybe it is superfluous?

Reply:
I suspect we do need /m, but I'll check; stay tuned!

Reply:
/m has no effect! You were right!